src/modules/_partials/layouts/DashboardLayout/index.tsx

```diff
@@ -55,15 +55,12 @@ const DashboardLayout = ({
   const { mode } = useTheme();
   return (
     <Flowbite theme={{ theme }}>
-      <div className="relative flex w-full overflow-x-hidden bg-brand-blue-light dark:text-gray-50 dark:bg-gray-900 h-screen overflow-y-auto">
+      <div className="relative flex w-screen overflow-x-scroll bg-brand-blue-light dark:text-gray-50 dark:bg-gray-900 h-screen overflow-y-auto">
         <AppSidebar className="hidden md:block md:w-fit" />
 
-        <div
-          id="main-content"
-          className="flex flex-col flex-grow h-full"
-        >
+        <div id="main-content" className="flex flex-col flex-grow">
           <AppNavbar />
-          <main className="flex flex-col w-full overflow-x-hidden bg-white flex-grow dark:bg-gray-900">
+          <main className="flex flex-col w-full overflow-x-auto bg-white flex-grow dark:bg-gray-900">
```

Comment:
Same as here, avoid horizontal scroll.

Reply:
@nicele08 I want to make the table responsive but be cause its parent overflow is hidden the table is being hidden as well do you have any suggestion on that?